### ISSUE TITLE
`NoSuchElementException` thrown when recording an XML view that contains Compose

### ISSUE THREAD

**Issue by mr-thierry** [cashapp/paparazzi#1704]
**Description** 
`NoSuchElementException` thrown when recording an XML and Compose view

**Steps to Reproduce**
I couldn't create a simple reproducible test. The view that is being screenshot is a combination of XML and Compose view. `AccessibilityRenderExtension` is used to render accessibility info of the view. 

Looking at the stacktrace, this should be easy to fix:

```
java.util.NoSuchElementException: List is empty.
	at kotlin.collections.CollectionsKt___CollectionsKt.first(_Collections.kt:221)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.processAccessibleChildren(AccessibilityRenderExtension.kt:119)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.processAccessibleChildren(AccessibilityRenderExtension.kt:144)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.processAccessibleChildren(AccessibilityRenderExtension.kt:144)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.processAccessibleChildren(AccessibilityRenderExtension.kt:103)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.processAccessibleChildren(AccessibilityRenderExtension.kt:108)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.processAccessibleChildren(AccessibilityRenderExtension.kt:108)
	at app.cash.paparazzi.accessibility.AccessibilityRenderExtension.renderView$lambda$5$lambda$4$lambda$3(AccessibilityRenderExtension.kt:73)
	at com.android.internal.view.OneShotPreDrawListener.onPreDraw(OneShotPreDrawListener.java:81)
	at android.view.ViewTreeObserver.dispatchOnPreDraw(ViewTreeObserver.java:1176)
	at android.view.AttachInfo_Accessor.dispatchOnPreDraw(AttachInfo_Accessor.java:58)
	at android.view.LayoutlibRenderer$1.onPreDraw(LayoutlibRenderer.java:45)
	at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:742)
	at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:798)
	at android.view.LayoutlibRenderer.draw(LayoutlibRenderer.java:41)
	at com.android.layoutlib.bridge.impl.RenderSessionImpl.renderAndBuildResult(RenderSessionImpl.java:584)
	at com.android.layoutlib.bridge.impl.RenderSessionImpl.render(RenderSessionImpl.java:446)
	at app.cash.paparazzi.PaparazziSdk.takeSnapshots$lambda$5(PaparazziSdk.kt:312)
	at app.cash.paparazzi.PaparazziSdk.withTime(PaparazziSdk.kt:372)
	at app.cash.paparazzi.PaparazziSdk.takeSnapshots(PaparazziSdk.kt:311)
	at app.cash.paparazzi.PaparazziSdk.snapshot(PaparazziSdk.kt:178)
	at app.cash.paparazzi.Paparazzi.snapshot(Paparazzi.kt:111)
	at app.cash.paparazzi.Paparazzi.snapshot$default(Paparazzi.kt:107)
...
```

**Expected behavior**
No exception is thrown and the test ends succesfully

**Additional information:**
- Paparazzi Version: 1.3.5
- OS: MacOS
- Compile SDK: Java 21
- Gradle Version: 8.10.2
- Android Gradle Plugin Version: 8.7.2

**Screenshots**
